src/components/App.js

```diff
@@ -272,7 +278,8 @@ export default class App extends React.Component {
     // If the volume is correctly set to the target, no need to change it
     if (currentVolume === this.state.audioConfig.targetVolume) {
       // If the audio is set to 0 and it’s been met, pause the audio
-      if (this.state.audioConfig.targetVolume === 0) this.pause();
+      if (this.state.audioConfig.targetVolume === 0 && this.state.pausing)
```

Comment:
So when would be the time that the vol is zero and we would not want to pause?

Reply:
If you're fading up then you shouldn't pause, even if the target volume is 0.  The idea is that most internet radios that I could find allowed you to toggle the play/pause state independently of the volume, and I thought we should provide a familiar UX.